examples/README.md

```diff
@@ -1,16 +1,26 @@
+## Prerequisites:
+
+Run the following command(s) to ensure proper functionality of the program. 
+
+```shell
+go mod tidy
```

Comment:
This makes no sense. We say to do go mod tidy but bellow we are actually running code from outside module scope (`./examples`) were added. :) We need to decide what we want people to do